README.md

```diff
@@ -12,12 +12,39 @@ För detta projektet krävs:
 
 ### Lokal dev miljö
 
-För att starta en lokal devmiljö där du kan redigera saker i strapi och testa sidorna i webapplikationen behöver du:
+För att starta en lokal devmiljö behöver du först klona projektet. Om [Docker Desktop](https://www.docker.com/products/docker-desktop/) används, starta denna applikation. Se till att det **inte** finns några docker images som kör på port 5050 och 5432. Kör därefter `docker compose up` i root för att starta en `nollesite` container.
 
-1. Starta CMS:et - `cd strapi` och `yarn dev`
-   - Detta kommer starta en lokal strapi instans på `http://localhost:1337`
-2. Starta Web - `cd web` och `yarn dev`
-   - Detta kommer starta en lokal webapplikation på `http://localhost:3000`
+Följande steg förklarar hur en strapi instans skapas, där du kan redigera nollningssidor. Därefter förklaras hur man skapar en lokal webapplikation för att testa nollningssidorna.
+
+#### ⚙ Strapi
+
+Skapa CMS:et på följande sätt:
+1. Skapa en `.env`-fil i `strapi/`
+   - Kopiera över innehållet från `.env.example`
+   - Fyll `.env`-filen med värden för `STRAPI_APP_KEYS`, `API_TOKEN_SALT` och `JWT_SECRET` från Bitwarden
+   - **OBS:** Använd ej `DB` variablerna från Bitwarden vid testning, då ändringar i så fall påverkar prod!
+   - > TODO: `SMTP` och `WIKI` variabler behövs ej för att få CMS:et att fungera. Men det används till ...
```

Comment:
SMTP_* variablerna behövs typ inte, men de används ifall man behöver skicka t.ex. "glömt lösenord" mail lokalt. Lättast är att skippa det i dev och ifall det skulle krisa så kan man ändra direkt i DBn.

Reply:
WIKI är nog obsolete nu också, hade en idé om att dra in ddgwiki guiden på förstasidan men efter wikiuppdateringen så slopade jag det.

Reply:
Uppdaterar det!